packages/core/src/components/List/List.tsx

```diff
@@ -0,0 +1,404 @@
+        checked={allSelected}
+      />
+    );
+  };
+
+  const renderItemText = (item) => {
+    const ItemText = wrapperTooltip(hasTooltips, item.label, item.label);
+
+    return !multiSelect && item.path ? (
+      <StyledLink
+        key={item.label}
+        route={item.path}
+        classes={{ a: clsx(listClasses.link, classes?.link) }}
+      >
+        <ItemText />
+      </StyledLink>
+    ) : (
+      <ItemText />
+    );
+  };
+
+  const renderMultiSelectItem = (item, itemId) => {
+    if (useSelector) {
+      const Selection = wrapperTooltip(
+        hasTooltips,
+        <StyledMultiSelectCheckBox
+          id={setId(itemId, "selector")}
+          label={item.label}
+          checked={item.selected}
+          disabled={item.disabled}
+          onChange={(evt) => handleSelect(evt, item)}
+          classes={{
+            root: clsx(listClasses.selectorRoot, classes?.selectorRoot),
+            container: clsx(
+              listClasses.selectorContainer,
+              classes?.selectorContainer
+            ),
+            label: clsx(listClasses.truncate, classes?.truncate),
+          }}
+        />,
+        item.label
+      );
+      return <Selection />;
+    }
+
+    return renderItemText(item);
+  };
+
+  const renderSingleSelectItem = (item, itemId) => {
+    if (useSelector) {
+      const Selection = wrapperTooltip(
+        hasTooltips,
+        <StyledSingleSelectRadio
+          id={setId(itemId, "selector")}
+          label={item.label}
+          checked={item.selected}
+          disabled={item.disabled}
+          classes={{
+            root: clsx(listClasses.selectorRoot, classes?.selectorRoot),
+            container: clsx(
+              listClasses.selectorContainer,
+              classes?.selectorContainer
+            ),
+            label: clsx(listClasses.truncate, classes?.truncate),
+          }}
+        />,
+        item.label
+      );
+      return <Selection />;
+    }
+    return renderItemText(item);
+  };
+
+  const renderListItem = (item, i, otherProps = {}) => {
+    const itemId = setId(id, "item", i);
+    const selected = item.selected || false;
+
+    let startAdornment = null;
+    if (!useSelector && item.icon) {
+      startAdornment = renderLeftIcon(item);
+    }
+
+    return (
+      <StyledListItem
+        key={i}
+        id={itemId}
+        role={selectable ? "option" : "menuitem"}
+        disabled={item.disabled || undefined}
+        className={clsx(listClasses.item, classes?.item)}
+        classes={{
+          selected: clsx(listClasses.itemSelector, classes?.itemSelector),
+        }}
+        selected={multiSelect || selected ? selected : undefined}
+        onClick={(evt) => handleSelect(evt, item)}
+        startAdornment={startAdornment}
+        endAdornment={
+          item.showNavIcon && (
+            <DropRightXS
+              className={clsx(listClasses.box, classes?.box)}
+              iconSize="XS"
+            />
```

Comment:
The [CSS from the box class](https://github.com/lumada-design/hv-uikit-react/blob/10621b6afb337e36ea171aae7c735beda015de30/packages/core/src/List/styles.js#L35) is missing in the component `DropRightXS`.